tests/test_shell_tools.py

```diff
@@ -0,0 +1,77 @@
+# SPDX-License-Identifier: MIT
+
+import unittest
+from argparse import Namespace
+
+import odxtools.cli.browse as browse
+import odxtools.cli.find as find
+import odxtools.cli.list as list
+
+
+class utilFunctions():
+
+    def list_arguments_and_execution(path_to_pdx_file='.\examples\somersault.pdx',
```

Comment:
IMO that name is at the same time too bit vague and too long. maybe:
```suggestion
    def run_list_tool(path_to_pdx_file='.\examples\somersault.pdx',
```